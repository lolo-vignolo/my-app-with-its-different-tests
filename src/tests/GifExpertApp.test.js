import { shallow } from 'enzyme';
import React from 'react';

import GifExpertApp from "../GifExpertApp.js"

describe("it will test <GitExpertApp", ()=>{

 test('reander de DOM', () => {

    const wrapper = shallow(<GifExpertApp /> )
    expect(wrapper).toMatchSnapshot();

     
 })
 


})