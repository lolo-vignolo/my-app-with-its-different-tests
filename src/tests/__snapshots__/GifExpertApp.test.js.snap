// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it will test <GitExpertApp reander de DOM 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    newCategory={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      Category="Peaky Blinders"
      key="Peaky Blinders"
    />
  </ol>
</Fragment>
`;
